src/test/java/org/broadinstitute/hellbender/GatkToolIntegrationTest.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.hellbender;
```

Comment:
This class should live in the same package as `GATKTool` (`org.broadinstitute.hellbender.engine`)